client/src/styles/legacy/_tree.scss

```diff
@@ -493,6 +493,22 @@
     ins {
       background-color: transparent;
       background-image: url(../images/sitetree_ss_default_icons.png);
+
+      &.font-icon-drag-handle {
+        background: none;
+        font-size: 1.15385rem; // 15px
+
+        &::before {
+          margin-left: -1px;
+        }
+      }
+
+      &.font-icon-right-dir {
+        visibility: hidden;
+        background: none;
+        font-size: 1.53846rem; // 20px
```

Comment:
Comment above says `font-size: 1.15385rem; // 15px`, is one of the two px values wrong?

Reply:
Ha, I missed that one is 1.**1**5 and the other is 1.5 🤦‍♂️